### ISSUE TITLE
Error: During class fetch: Uncaught ReflectionException

### ISSUE THREAD

**Issue by jspizziri** [api-platform/api-platform#1207]
Hello,

I'm attempting to create a custom controller action/endpoint for a record upsert. I'm following the documentation [here](https://api-platform.com/docs/core/operations/#creating-custom-operations-and-controllers) and I'm on api-platform@2.4.5.

### Exception

```
Error: During class fetch: Uncaught ReflectionException: Class Symfony\Component\Form\FormTypeGuesserInterface not found in /srv/api/vendor/symfony/doctrine-bridge/Form/DoctrineOrmTypeGuesser.php:25
Stack trace:
#0 /srv/api/vendor/symfony/debug/DebugClassLoader.php(160): require('/srv/api/vendor...')
#1 [internal function]: Symfony\Component\Debug\DebugClassLoader->loadClass('Symfony\\Bridge\\...')
#2 [internal function]: spl_autoload_call('Symfony\\Bridge\\...')
#3 /srv/api/vendor/symfony/config/Resource/ClassExistenceResource.php(78):class_exists('Symfony\\Bridge\\...')
#4 /srv/api/vendor/symfony/dependency-injection/ContainerBuilder.php(353):Symfony\Component\Config\Resource\ClassExistenceResource->isFresh(0)
#5 /srv/api/vendor/symfony/dependency-injection/Compiler/AutowirePass.php(336): Symfony\Component\DependencyInjection\ContainerBuilder->getReflectionClass('Symfony\\Bridge\\...', false)
#6 /srv/api/vendor/symfony/dependency-injection/Compiler/AutowirePass.php(322): Symfony\Component\DependencyInjection\Com
```

### Code

```php
<?php
namespace App\Controller;

use App\Entity\FooBar;

class FooBarsUpsert
{
    // the typehinting of "FooBar" here is causing symfony to attempt to inject the entity,
    // resulting in the exception
    public function __invoke(FooBar $data): FooBar
    {
        return $data;
    }
}
```

```php
<?php

namespace App\Entity;

use App\Controller\FooBarUpsert;
use ApiPlatform\Core\Annotation\ApiResource;
use Doctrine\ORM\Mapping as ORM;
use Symfony\Component\Validator\Constraints as Assert;


/**
 * @ORM\Entity
 * @ApiResource(
 *     itemOperations={
 *         "upsert"={
 *             "method"="PUT",
 *             "path"="/foo_bar/{id}",
 *             "controller"=FooBarUpsert::class,
 *             "read"=false,
 *         }
 *     }
 * )
 */
class FooBar
{
    /**
     * @var int The entity Id
     *
     * @ORM\Id
     * @ORM\GeneratedValue
     * @ORM\Column(type="integer")
     */
    public $id;
}
```

Thanks in advance!

**Comment by teohhanhui** [comment]
This is https://github.com/symfony/symfony/issues/32395

**Comment by jspizziri** [comment]
@teohhanhui thanks for the comment. I had found that, however I'm using the standard Docker & docker-compose provided in the app itself. If the problem is the PHP version then I'd imagine that the docker build needs to be updated? Or am I incorrect.

**Comment by teohhanhui** [comment]
The problem will fix itself when PHP 7.3.8 is released. :smile:
